certs/ca-cert.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
+MIIDrTCCApWgAwIBAgIUEPJAxDYmRJ2mMpXba8rAG1CsYbQwDQYJKoZIhvcNAQEL
+BQAwZjELMAkGA1UEBhMCVVMxDjAMBgNVBAgMBVN0YXRlMQ0wCwYDVQQHDARDaXR5
+MRUwEwYDVQQKDAxPcmdhbml6YXRpb24xDTALBgNVBAsMBFVuaXQxEjAQBgNVBAMM
+CWxvY2FsaG9zdDAeFw0yNDEwMDcxNzMzMzhaFw0yNTEwMDcxNzMzMzhaMGYxCzAJ
+BgNVBAYTAlVTMQ4wDAYDVQQIDAVTdGF0ZTENMAsGA1UEBwwEQ2l0eTEVMBMGA1UE
+CgwMT3JnYW5pemF0aW9uMQ0wCwYDVQQLDARVbml0MRIwEAYDVQQDDAlsb2NhbGhv
+c3QwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCw5Zv1Ya083pPRIyr2
+jP/DEWUoYhQJqV36gzn25ZVdFyFQ7+/tJ+xjK11bK+VjN7OQHbB4xft4Io2qbfZR
+1/YPQReXvpq+Aa+nVMZ2cSHJ9/pddLKu39+4dYLtwCDNm8EM9eikxQ0NJq7OO3sh
+UyKZhbxdXfcZ/jD7oOf90FzkWlYZzNt3NGDafZ2cyCvzavepxiTazaw03q4u0cLm
+fKL9ZJUNxpxTA/Pky3oNTYJTPbwRaf/i1oWkZUzpyqEnE98DuXQ31eGbhOaMR4k2
+AjjUMWF4euU3Y+WYj/vaC2v7CMVbyhipeSFSk2DjifKJHZicsFQ2nmec6LDDtFBb
+4DSlAgMBAAGjUzBRMB0GA1UdDgQWBBRjyBAujDf2/IXpKyNtsKsWTxRybDAfBgNV
+HSMEGDAWgBRjyBAujDf2/IXpKyNtsKsWTxRybDAPBgNVHRMBAf8EBTADAQH/MA0G
+CSqGSIb3DQEBCwUAA4IBAQBFo4COyIMMDUSF2y2BvqmoF7ZdkrwuoJfV0HZ3BufR
+Uy/8LbjBAAfzwcY0wlk/emmKTos5vsFKfLPOMrGKMPuW6EgTddjubZQ4YXwzxkbR
+S7vvqzp2Beg5OGsYV9B/4RX5jbGmxrlNpRivFtMceQ3toi3Kyg2MEnu8UcL193ln
+Do6L/Jwv7TWV5I5SW3B5gyaoA5w5sgx0DOdoicxR4MkvS4FL44QxdUV4Y91aNs8e
+qO6kh8RCLC2P7t6ecQ/I2VEOCQB+CgVA5rnGjbVXzS3YHvDLZ0LBJgdFgzxURBny
+6a7DoL2M/Oq8ZtKNGMk7EuVq2jMok6kMMi1ZBvSwlErd
+-----END CERTIFICATE-----
```

Comment:
Not blocking and won't affect anything because we run it locally, but best if we added `certs/` to `.gitignore`. I can make a fast follow to add this after you merge

Reply:
sounds good